ros/angel_system_nodes/angel_system_nodes/spatial_mapper.py

```diff
@@ -351,12 +328,14 @@ def convert_pixel_coord_to_world_coord(self, world_matrix_2d,
 
         scaled_point = self.scale_pixel_coordinates(p)
 
-        focal_length_x = projection_matrix[0][0]
-        focal_length_y = projection_matrix[1][1]
-        center_x = projection_matrix[0][2]
-        center_y = projection_matrix[1][2]
+        #focal_length_x = projection_matrix[0][0]
+        focal_length_x = 1.63
```

Comment:
Now that I'm seeing this set multiple times, the value should probably be set as a global constant that is used in both places. Hack commentary could below with those global defs.